docs/content/concepts/sui-move-concepts/packages.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Package Upgrades
+title: Packages
```

Comment:
This section talks about more than just upgrades and just naming it `Packages` seems fitting, there is already an `Upgrading Packages` subpage